Sources/SwiftSDKGenerator/Generator/LocalSwiftSDKGenerator.swift

```diff
@@ -344,6 +344,21 @@ public final class LocalSwiftSDKGenerator: SwiftSDKGenerator {
     }
   }
 
+  public func buildCMakeProject(_ projectPath: FilePath) async throws -> FilePath {
+    try await Shell.run(
```

Comment:
It's fine to me to use Shell based command invocation for now, but do you have any plan to switch to Process based invocation to avoid unintentional shell parameter expansion?

Reply:
Yes, I would love to, unfortunately `ProcessExecutor` from the `AsyncProcess` module here doesn't support the `currentWorkingDirectory` API.

Reply:
We would need to evaluate our use of `currentWorkingDirectory` first, and check if there's a way to avoid any uses of it. Should be possible for both `cmake` and `swift build` invocations, but there are a few more commands remaining that I haven't checked yet.